tests/unit_tests/pyanaconda_tests/test_argparse.py

```diff
@@ -165,6 +165,51 @@ def test_target(self):
         assert conf.target.is_directory == True
         assert conf.target.physical_root == "/what/ever"
 
+    def test_target_nosave(self):
+        conf = AnacondaConfiguration.from_defaults()
+
+        opts, _removed = self._parseCmdline([])
+        conf.set_from_opts(opts)
+
+        assert conf.target.copy_input_kickstart == True
```

Comment:
I think you need to use `is` instead of `==`.

Reply:
That's what the conversion script gave us, and what is in the rest of the tests. I think it would use `is` if it was a better choice?

The `assert` statement takes a `bool`, therefore the comparisons are redundant - for `True` I'd just leave the value, for `False` add a `not`. That's how one should assert, ideally. Is that acceptable?

Reply:
Ah, yes, `is` seems to be correct.